procedures: Add PostgreSQL migration guides (#2163)
codenvy/che-docs: modules/administration-guide/partials/assembly_migration-to-external-database-setup.adoc

:parent-context-of-migration-to-external-database-setup: {context}

> [id="migration-to-external-database-setup_{context}"]
> = Migration to external database setup

Reviewer: ```suggestion
[id="migration-to-an-external-database-setup_{context}"]
= Migration to an external database setup
```

Reviewer: You'll probably need to change the file name too.

:context: migration-to-external-database-setup

> Follow the procedures below to migrate from internal to external database setup successfully without any data loss.

Reviewer: ```suggestion
You can migrate from the internal database to an external database without losing data.
```
